modules/backend/widgets/Lists.php

```diff
@@ -1415,6 +1415,7 @@ public function setSearchTerm($term)
             $this->showTree = false;
         }
 
+        $this->currentPageNumber = 1;
```

Comment:
@Samuell1 does setting this to page 1 here also change `lastVisitedPage` in the session to page 1 as well? If it doesn't, it'll need to, otherwise a refresh would take them to a potentially different page.

Reply:
Found out that pagination needs to be totally reworked somehow because this PR still doesnt fix that issue. I think you are true we need to save lastVisitedPage on every call where pagination chages because now its only saved when prepareVars method is called.

Reply:
@bennothommo Found a fix! PR is now updated.